docs/UserGuide.md

```diff
@@ -131,9 +142,12 @@ Format: `addstudent -n <student_name> -i <student_id> [-g <group_name>]... [-t <
 * There should not be an existing student with the same NUSNET ID. If there is, the student to be added is considered invalid.
 
 Examples:
-* `addstudent -n Jonas Chow -i E0123456` adds the student Jonas Chow with the given NUSNET ID.
-* `addstudent -n Jonas Chow -i E0123456 -g T01A -g R01A` adds the student Jonas Chow and allocate him into groups `T01A` and `R01A`.
-* `addstudent -n Jonas Chow -i E0123456 -t beginner` adds the student Jonas Chow and tag him with `beginner`.
+* `addstudent -n Jonas Chow -i E0123456` 
+  * adds the student Jonas Chow with the given NUSNET ID.
+* `addstudent -n Jonas Chow -i E0123456 -g T01A -g R01A` 
+  * adds the student Jonas Chow and allocate him into groups `T01A` and `R01A`.
+* `addstudent -n Jonas Chow -i E0123456 -t beginner` 
+  * adds the student Jonas Chow and tag him with `beginner`.
```

Comment:
Shouldn't "allocate" and "tag" be "allocates" and "tags" to be consistent with "adds"?

```suggestion
  * adds the student Jonas Chow and allocates him into groups `T01A` and `R01A`.
* `addstudent -n Jonas Chow -i E0123456 -t beginner` 
  * adds the student Jonas Chow and tags him with `beginner`.
```